python_ta/contracts/__init__.py

```diff
@@ -40,6 +40,11 @@
 In most cases you should not need to change this!
 """
 
+STRICT_NUMERIC_TYPES = True
+"""
+Set to False to allow implicit conversions between numeric types (i.e, an int can be used where a float is expected).
```

Comment:
It's a bit too broad to say "implicit conversions between numeric types" because the conversions can only go one way (e.g., a `float` can't be accepted as an `int`).

You can reword this to just "to allow more specific numeric types to be accepted by more general type annotations", similar to the above comment